### PR TITLE
chore: changelog updates for v49.11.0

### PR DESCRIPTION
**Pull request by sfsholden**
### What does this PR do?
Change log updates for 49.11.0.

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# 49.11.0 - September 23, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-apex
+
+- Update apex syntax highlighting for Safe Navigation Operator ([PR #2557](https://github.com/forcedotcom/salesforcedx-vscode/pull/2557))
```
Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: Capitalize the Apex in "Apex syntax highlighting

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+
+#### docs
+
+- Update [Dreamforce Session Videos](https://developer.salesforce.com/tools/vscode/#dreamforce-session-videos) in documentation site ([PR #2537](https://github.com/forcedotcom/salesforcedx-vscode/pull/2537))
```
Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:11]: "on" documentation site? "on" sounds better to me. :)

**Review by ekapner — COMMENTED** [review]
A few very minor changes

**Review by rcoringrato-sfdc — APPROVED** [review]

**Review by ekapner — APPROVED** [review]
Looks good!

**Pull request merged**